### PR TITLE
Keypad: First commit

### PR DESCRIPTION
**Pull request by simoneduca**
Setting up the keypad for abbreviations.

### CODE REVIEW

File: app/modules/transcribe/overlay/keypad-slide.directive.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            if (start === end) {
+                textAfter = text.substring(start, text.length);
+                textarea.val(textBefore + startTag + endTag + textAfter);
+            } else {
+                textInBetween = text.substring(start, end);
+                textAfter = text.substring(end, text.length);
+                textarea.val(textBefore + startTag + textInBetween + endTag + textAfter);
+            }
+            var pos = startTag.length + text.length;
+            console.log(pos);
```
Comment (rogerhutchings) [inline app/modules/transcribe/overlay/keypad-slide.directive.js]: I'd remove this

File: app/modules/transcribe/overlay/keypad-slide.directive.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            textarea.caret(pos);
```
Comment (rogerhutchings) [inline app/modules/transcribe/overlay/keypad-slide.directive.js]: You're only using `pos` here, so no point in having a separate variable for it; just do `textarea.caret(startTag.length + text.length);`

File: app/styles/components/keypad-slide.styl [hunk 3]
```diff
@@ -0,0 +1,52 @@
+.keypad-slide
+    @extend .panel
+    @extend .panel-default
+    width: 25%
```
Comment (rogerhutchings) [inline app/styles/components/keypad-slide.styl]: Just for simplicity, I'd order the properties alphabetically

Comment (simoneduca) [inline app/styles/components/keypad-slide.styl]: Sure

File: app/modules/transcribe/overlay/keypad-slide.directive.js (hunk 2 above)
Comment (simoneduca) [inline app/modules/transcribe/overlay/keypad-slide.directive.js]: Done

File: app/modules/transcribe/overlay/keypad-slide.directive.js (hunk 1 above)
Comment (simoneduca) [inline app/modules/transcribe/overlay/keypad-slide.directive.js]: Done

**Pull request merged**